app/models/agent_builder.rb

```diff
@@ -0,0 +1,18 @@
+class AgentBuilder
+  class << self
+    def build_for_type(type, user, attributes = {})
+      begin
+        klass = Agents.const_get(type)
```

Comment:
Agreed. It no longer seems to offer much value. Sounds like no objections if I pull it out completely?